.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -2,23 +2,16 @@
 name: Enhancement
 about: Request for a feature to be added to an existing component.
 title: "Enhancement: "
-labels: enhancement
+labels: enhancement, 0 - new
 assignees: ""
 ---
 
 ### Description
 
 ### Acceptance Criteria <!--(a.k.a. Requirements)-->
 
-#### Desired New Behavior
```

Comment:
Nitpick: I think moving `Desired X` to a comment on `Acceptance Criteria` could be useful.